Patches/Combat Enthusiast's Collection/ThingDefs_Weapons.xml

```diff
@@ -55,13 +55,6 @@
 					<researchPrerequisite>PrecisionRifling</researchPrerequisite>
 				</li>
 
```

Comment:
positively remember irl names with CE was a feature in the original version of the mod